capitalize_deco.rb

```diff
@@ -0,0 +1,8 @@
+require './base_deco'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    capitalized = @nameable.correct_name.upcase
+    p capitalized
```

Comment:
- [ ] The correct_name method should return a capitalized name, meaning it should capitalize only the first letter of the name not make it uppercase. Kindly make sure to fix this so your code work as expected, you should use `.upercase` instead